map/read-geopackage/src/main/java/com/esri/samples/read_geopackage/ReadGeoPackageSample.java

```diff
@@ -52,11 +54,18 @@ public void start(Stage stage) {
       stage.setScene(scene);
       stage.show();
 
+      // authentication with an API key or named user is required to access basemaps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
       // create a map and add it to the map view
-      final ArcGISMap map = new ArcGISMap(Basemap.Type.STREETS, 39.7294, -104.8319, 11);
+      final ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_STREETS);
       mapView = new MapView();
       mapView.setMap(map);
 
+      // set a viewpoint on the map view
+      mapView.setViewpoint(new Viewpoint(39.7294, -104.8319, 300000.0));
```

Comment:
for LOD 11, we could go for 288895, rounded down to 288800 should be ok if you want it to look a little neater!